files/chrono/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "chrono",
+  "repo": "wanasit/chrono",
+  "files": {
+    "include": ["chrono.js", "chrono.min.js", "chrono.map.json"]
```

Comment:
I would say that anyone who is looking to debug the file, would have the knowledge to find your github page and look through the un-minified file there. But I am not affiliated with jsdelivr, so maybe they have different views about it.